.travis.yml

```diff
@@ -9,16 +10,16 @@ before_script:
   - go get github.com/modocache/gover
   - go get github.com/mattn/goveralls
   - go get -v github.com/GeertJohan/fgt
+
 script:
   - go get github.com/cloudflare/gokeyless/...
-  - go test github.com/cloudflare/gokeyless/...
   - go vet github.com/cloudflare/gokeyless/...
   - fgt golint github.com/cloudflare/gokeyless/...
-  - go list -f '{{if len .TestGoFiles}}"go test -coverprofile={{.Dir}}/.coverprofile {{.ImportPath}}"{{end}}' ./... | xargs -i sh -c {}
-  - gover
-  - if [ "$COVERALLS_TOKEN" != "" ]; then
-      goveralls -coverprofile=gover.coverprofile -service=travis-ci -repotoken $COVERALLS_TOKEN;
-    fi
+  - go test -coverprofile=coverage.txt -covermode=atomic ./tests/
```

Comment:
Code coverage isn't working as the tests aren't in the same directory as the source files. (They can't really be, because there's shared server code between all of them)